lib/login/view/sign_up_page.dart

```diff
@@ -135,13 +136,15 @@ class _SignUpState extends State<SignUp> {
                   ),
                   SizedBox(height: 30),
                   TextFormField(
+                    autovalidateMode: AutovalidateMode.onUserInteraction,
                     onChanged: (value) {
                       emailValue = value;
                     },
                     validator: (value) {
                       if (value == null || value.isEmpty) {
                         return AppLocalizations.of(context)!.emptyEmail;
-                      } else if (validateEmail(emailValue) == false) {
+                      } else if (ValidationUtils.validateEmail(emailValue) ==
```

Comment:
instead of " if (ValidationUtils.validateEmail(emailValue) == false) please use => 
If(!ValidationUtils.validateEmail(emailValue))